build/build.go

```diff
@@ -252,17 +252,22 @@ func toSolveOpt(ctx context.Context, node builder.Node, multiDriver bool, opt Op
 			attests[k] = *v
 		}
 	}
-	supportsAttestations := bopts.LLBCaps.Contains(apicaps.CapID("exporter.image.attestations")) && nodeDriver.Features(ctx)[driver.MultiPlatform]
+
+	supportAttestations := bopts.LLBCaps.Contains(apicaps.CapID("exporter.image.attestations")) && nodeDriver.Features(ctx)[driver.MultiPlatform]
 	if len(attests) > 0 {
-		if !supportsAttestations {
-			return nil, nil, errors.Errorf("attestations are not supported by the current buildkitd")
+		if !supportAttestations {
+			if !nodeDriver.Features(ctx)[driver.MultiPlatform] {
+				return nil, nil, errors.Errorf(`Attestations are not supported for the %s driver.
+Switch to a different driver, or turn on the containerd image store, and try again.`, nodeDriver.Factory().Name())
```

Comment:
@dvdksn for copy :pray:

Reply:
Maybe use the `notSupported` func?

https://github.com/docker/buildx/blob/f258de4c853460f6035a98d778db5cef12c32c36/build/build.go#L1429-L1433

Reply:
It's referred to in the note:

<img width="824" alt="image" src="https://github.com/docker/buildx/assets/35727626/06f3290c-451c-46af-81ad-f37812316321">

Reply:
I can't read :man_shrugging: